.github/workflows/ci.yml

```diff
@@ -91,6 +91,9 @@ jobs:
             - name: Build other binaries
               run: cargo build --workspace --bins --all-features
 
+            - name: Build Blade version
```

Comment:
I don't think we'd want to add this to CI yet.

Reply:
Could we at least test `gpui` alone with this feature on CI (as opposed to full Zed)?

Reply:
We've decided to keep it in CI, to make sure this is maintained :)